content/src/content/jcr_root/apps/core/wcm/components/tabs/v1/tabs/clientlibs/site/js/tabs.js

```diff
@@ -68,6 +68,8 @@
          * @param {TabsConfig} config The Tabs configuration
          */
         function init(config) {
+            that._config = config;
```

Comment:
where do you use `that._config`?

Reply:
it's used in the utils: `getDeepLinkItemIdx`